### DISCUSSION TITLE
Can't select an OpenKeyChain key

### DISCUSSION THREAD

**Discussion by ook** [zeapo/Android-Password-Store#2685] [Q&A] [answered]
Hi,

My goal is to load my gopass store into PasswordStore. I generated a new key in OpenKeyChain, but when I ask to clone my repository using the key, I can't select it in the modal:

![photo_2023-09-16_00-09-56](https://github.com/android-password-store/Android-Password-Store/assets/559/270898d5-a5ba-4121-aa0b-eee1e376836d)

Why my OpenkeyChain key doesn't appear here?

Thank you.

**Comment by msfjarvis** [comment, marked as answer]
It doesn't have an authentication subkey, you'll have to ask the OpenKeychain folks how to add it. You'll also need to add the derived SSH key to your Git server before it can be used to clone the repo.

**Comment by ook** [comment]
Oh, thank you @msfjarvis : I added an auth subkey (from the advanced menu) and it works.
